app/models/user.rb

```diff
@@ -1,5 +1,6 @@
 class User < ActiveRecord::Base
   has_secure_password
+  validates_confirmation_of :password
```

Comment:
Prefer the new style validations `validates :column, confirmation: value` over `validates_confirmation_of`.